### ISSUE TITLE
Kneeling Results

### ISSUE THREAD

**Issue by redradman** [redradman/MujocoPoseLearning#24]
Hyperparams and details in commit 175bbb7
train time: 5,000,000
bash output: 
```bash
-----------------------------------------
| reward/                 |             |
|    max_episode          | 1.16e+03    |
|    mean_episode         | 116         |
|    min_episode          | 0           |
| time/                   |             |
|    fps                  | 6312        |
|    iterations           | 306         |
|    time_elapsed         | 794         |
|    total_timesteps      | 5013504     |
| train/                  |             |
|    approx_kl            | 0.045490645 |
|    clip_fraction        | 0.396       |
|    clip_range           | 0.2         |
|    entropy_loss         | -22.2       |
|    explained_variance   | 0.97        |
|    learning_rate        | 0.0001      |
|    loss                 | 0.105       |
|    n_updates            | 3050        |
|    policy_gradient_loss | -0.0172     |
|    std                  | 0.709       |
|    value_loss           | 0.561       |
```
Reward function: `robust_standing_reward`

Videos: 

https://github.com/user-attachments/assets/b1a1742d-17b8-466e-a786-9ebe91855199

https://github.com/user-attachments/assets/2ebba3fc-6531-437e-a70b-4e62d88b26e9

https://github.com/user-attachments/assets/e4a11381-6b09-4b0b-961c-d569c14cb372

https://github.com/user-attachments/assets/51f1a732-fc8f-4918-a87d-6f51efd623ac

https://github.com/user-attachments/assets/cdf83fac-f68a-4d7d-a5a3-c8565f3e2cdb

https://github.com/user-attachments/assets/26e46ac3-a631-4e39-b790-cf80dcc1e85c
